src/main/kotlin/app/revanced/patches/youtube/misc/openlinksexternally/patch/OpenLinksExternallyPatch.kt

```diff
@@ -0,0 +1,63 @@
+package app.revanced.patches.youtube.misc.openlinksexternally.patch
+
+import app.revanced.extensions.toErrorResult
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstructions
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.shared.settings.preference.impl.StringResource
+import app.revanced.patches.shared.settings.preference.impl.SwitchPreference
+import app.revanced.patches.youtube.misc.openlinksexternally.annotations.OpenLinksExternallyCompatibility
+import app.revanced.patches.youtube.misc.openlinksexternally.fingerprints.*
+import app.revanced.patches.youtube.misc.settings.bytecode.patch.SettingsPatch
+import org.jf.dexlib2.iface.instruction.formats.Instruction21c
+
+@Patch
+@Name("open-links-externally")
+@Description("Open links outside of the app directly in your browser.")
+@OpenLinksExternallyCompatibility
+@Version("0.0.1")
+class OpenLinksExternallyPatch : BytecodePatch(
+    listOf(
+        OpenLinksExternallyPrimaryFingerprint,
+        OpenLinksExternallySecondaryFingerprint,
+        OpenLinksExternallyTertiaryFingerprint
```

Comment:
Please name the fingerprint according to what the methods do. Naming them properly allows for easier maintenance in the future.